### ISSUE TITLE
Can not call create-board / create-template / query-board from CLI

### ISSUE THREAD

**Issue by Boris-Scharinger** [Collaborne/miro-templating#1]
Hey, I followed the instructions, however it would not access / find the scripts as, e.g. MIRO_TOKEN=<> BOARD_ID=<> TEMPLATE_ID=<> create-miro-template
It tells me it can't identify the expression as a script, CMDlet or such...
Any idea, what I would need to do different?

**Comment by Boris-Scharinger** [comment]
It seems, you were not developing under Windows, correct? I seem to have it running now...but maybe it makes sense to update the documentation?

**Comment by ronnyroeller** [comment]
Boris - great to hear that you already figured out how to make it run.
Unfortunately, I don't have a Windows system here to try it out.
You should be able to run the npm scripts directly like this example:

```
MIRO_TOKEN=<TOKEN> BOARD_ID=<BOARD_ID> TEMPLATE_ID=<TEMPLATE_ID> npm run create-template
```

**Comment by Boris-Scharinger** [comment]
Hi Ronny, it continues to be tricky:
1) I installed miro-templating without my console in 'Admin Mode' that's why npm install **-g** miro-templating wasn't executed properly.
2) It seems, cmd and ps1 files have been generated too (that wasn't clear in the first place). But it terms of how you call them with the parameters is unclear (to me). Don't worry, I will find out.
3) Yes, you are right, we will use the js/ typescript-implementation directly. Just getting my head around how that will look like in the mendix environment.

The scripts were meant to provide a fast PoC entry point which is great. But then it turned out to be trickier than I thought. I will keep you posted.

**Comment by ronnyroeller** [comment]
> I installed miro-templating without my console in 'Admin Mode' that's why npm install -g miro-templating wasn't executed properly.

Indeed, depending on your setup, `npm` likely needs admin permissions for a global installation.

> It seems, cmd and ps1 files have been generated too (that wasn't clear in the first place). But it terms of how you call them with the parameters is unclear (to me). Don't worry, I will find out.

The parameters are environment variables.
I think you can set them on the Windows command line via:
```
set MIRO_TOKEN=SOMETHING
echo %MIRO_TOKEN%
```

**Comment by ankon** [comment]
> > The parameters are environment variables.
> > I think you can set them on the Windows command line via:
> > 
> > ```cmd
> > set MIRO_TOKEN=SOMETHING
> > echo %MIRO_TOKEN%
> > ```

Indeed. 

@Boris-Scharinger I opened https://github.com/Collaborne/miro-templating/pull/2, could you maybe have a look at see whether those changes help for Windows, or whether you needed additional changes?
